### PR TITLE
Docs: Make `identity` and `zones` documentation consistent

### PR DESCRIPTION
**Pull request by stephybun**

### CODE REVIEW

File: website/docs/d/data_protection_backup_vault.html.markdown [hunk 1]
```diff
@@ -51,11 +51,13 @@ output "azurerm_data_protection_backup_vault_principal_id" {
 
 `identity` exports the following:
```
Comment (jackofallops) [inline website/docs/d/data_protection_backup_vault.html.markdown:52]: should this line be removed?

File: website/docs/r/data_share_account.html.markdown [hunk 2]
```diff
@@ -57,7 +57,7 @@ The following arguments are supported:
 
 An `identity` block supports the following:
 
-* `type` - (Required) Specifies the identity type of the Data Share Account. At this time the only allowed value is `SystemAssigned`. Changing this forces a new resource to be created.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Data Share Account. Only possible value is `SystemAssigned`.
```
Comment (jackofallops) [inline website/docs/r/data_share_account.html.markdown:60]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Data Share Account. The only possible value is `SystemAssigned`.
```

File: website/docs/r/eventhub_namespace.html.markdown [hunk 3]
```diff
@@ -63,7 +63,7 @@ The following arguments are supported:
 
 A `identity` block supports the following:
 
-* `type` - (Required) The Type of Identity which should be used for this EventHub Namespace. At this time the only possible value is `SystemAssigned`.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Event Hub Namespace. Only possible value is `SystemAssigned`.
```
Comment (jackofallops) [inline website/docs/r/eventhub_namespace.html.markdown:66]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Event Hub Namespace. The only possible value is `SystemAssigned`.
```

File: website/docs/r/express_route_port.html.markdown [hunk 4]
```diff
@@ -56,11 +56,11 @@ The following arguments are supported:
 
 ---
 
-A `identity` block supports the following:
+An `identity` block supports the following:
 
-* `type` - (Required) The type of the identity used for the Express Route Port. Currently, the only possible values is `UserAssigned`.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Express Route Port. Only possible value is `UserAssigned`.
```
Comment (jackofallops) [inline website/docs/r/express_route_port.html.markdown:61]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Express Route Port. The only possible value is `UserAssigned`.
```

File: website/docs/r/firewall_policy.html.markdown [hunk 5]
```diff
@@ -71,9 +71,9 @@ A `dns` block supports the following:
 
 A `identity` block supports the following:
 
-* `type` - (Required) The Type of Identity which should be assigned to this Firewall Policy. At this time the only supported value is `UserAssigned`. Changing this forces a new Firewall Policy to be created.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Firewall Policy. Only possible value is `UserAssigned`.
```
Comment (jackofallops) [inline website/docs/r/firewall_policy.html.markdown:74]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Firewall Policy. The only possible value is `UserAssigned`.
```

File: website/docs/r/mysql_server.html.markdown [hunk 6]
```diff
@@ -90,8 +90,8 @@ The following arguments are supported:
 ---
 
 A `identity` block supports the following:
-   
-* `type` - (Required) The Type of Identity which should be used for this MySQL Server. At this time the only possible value is `SystemAssigned`.
+
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this MySQL Server. Only possible value is `SystemAssigned`.
```
Comment (jackofallops) [inline website/docs/r/mysql_server.html.markdown:94]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this MySQL Server. The only possible value is `SystemAssigned`.
```

File: website/docs/r/purview_account.html.markdown [hunk 7]
```diff
@@ -55,7 +55,7 @@ The following arguments are supported:
 
 The `identity` block supports the following:
 
-* `type` - (Required) The Type of Managed Identity to assign to this Purview Account. At this time the only possible value is `SystemAssigned`. Changing this forces a new resource to be created.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Purview Account. Only possible value is `SystemAssigned`.
```
Comment (jackofallops) [inline website/docs/r/purview_account.html.markdown:58]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Purview Account. The only possible value is `SystemAssigned`.
```

File: website/docs/r/recovery_services_vault.html.markdown [hunk 8]
```diff
@@ -57,7 +58,7 @@ The following arguments are supported:
 
 An `identity` block supports the following:
 
-* `type` - (Required) The Type of Identity which should be used for this Recovery Services Vault. At this time the only possible value is `SystemAssigned`.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Recovery Services Vault. Only possible value is `SystemAssigned`.
```
Comment (jackofallops) [inline website/docs/r/recovery_services_vault.html.markdown:61]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Recovery Services Vault. The only possible value is `SystemAssigned`.
```

File: website/docs/r/search_service.html.markdown [hunk 9]
```diff
@@ -62,7 +62,7 @@ The following arguments are supported:
 
 A `identity` block supports the following:
 
-* `type` - (Required) The Type of Identity which should be used for the Search Service. At this time the only possible value is `SystemAssigned`.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Search Service. Only possible value is `SystemAssigned`.
```
Comment (jackofallops) [inline website/docs/r/search_service.html.markdown:65]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Search Service. The only possible value is `SystemAssigned`.
```

File: website/docs/r/spring_cloud_app.html.markdown [hunk 10]
```diff
@@ -78,7 +78,7 @@ An `custom_persistent_disk` block exports the following:
 
 An `identity` block supports the following:
 
-* `type` - (Required) Specifies the identity type of the Spring Cloud Application. Possible value is `SystemAssigned`.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Cosmos Account. Only possible value is `SystemAssigned`.
```
Comment (jackofallops) [inline website/docs/r/spring_cloud_app.html.markdown:81]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Cosmos Account. The only possible value is `SystemAssigned`.
```

File: website/docs/r/sql_managed_instance.html.markdown [hunk 11]
```diff
@@ -254,7 +254,7 @@ A `sku` block supports the following:
 
  An `identity` block supports the following:
 
- * `type` - (Required) The identity type of the SQL Managed Instance. Only possible values is `SystemAssigned`.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this SQL Managed Instance. Only possible value is `SystemAssigned`.
```
Comment (jackofallops) [inline website/docs/r/sql_managed_instance.html.markdown:257]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this SQL Managed Instance. The only possible value is `SystemAssigned`.
```

File: website/docs/r/sql_server.html.markdown [hunk 12]
```diff
@@ -73,7 +73,7 @@ The following arguments are supported:
 
 An `identity` block supports the following:
 
-* `type` - (Required) Specifies the identity type of the Microsoft SQL Server. At this time the only allowed value is `SystemAssigned`.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this SQL Server. Only possible value is `SystemAssigned`.
```
Comment (jackofallops) [inline website/docs/r/sql_server.html.markdown:76]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this SQL Server. The only possible value is `SystemAssigned`.
```

File: website/docs/r/stream_analytics_job.html.markdown [hunk 13]
```diff
@@ -81,7 +81,7 @@ The following arguments are supported:
 
 An `identity` block supports the following:
 
-* `type` - (Required) The type of identity used for the Stream Analytics Job. The only possible value is `SystemAssigned`.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Stream Analytics Job. Only possible value is `SystemAssigned`.
```
Comment (jackofallops) [inline website/docs/r/stream_analytics_job.html.markdown:84]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Stream Analytics Job. The only possible value is `SystemAssigned`.
```

File: website/docs/r/synapse_workspace.html.markdown [hunk 14]
```diff
@@ -253,7 +253,7 @@ A `customer_managed_key` block supports the following:
 
 The `identity` block supports the following:
 
-* `type` - (Required) the Type of Managed Identity to assign to this Synapse Workspace. At this time the only supported value is `SystemAssigned`. Changing this forces a new resource to be created.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Synapse Workspace. Only possible value is `SystemAssigned`.
```
Comment (jackofallops) [inline website/docs/r/synapse_workspace.html.markdown:256]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Synapse Workspace. The only possible value is `SystemAssigned`.
```

File: website/docs/r/video_analyzer.html.markdown [hunk 15]
```diff
@@ -94,15 +94,15 @@ A `storage_account` block supports the following:
...
 
-* `user_assigned_identity_id` - (Required) Specifies the User Assigned Identity ID which should be assigned to a access this Storage Account.
+* `user_assigned_identity_id` - (Required) Specifies the User Assigned Identity ID which should be assigned to access this Storage Account.
 
 ---
 
 A `identity` block supports the following:
 
-* `type` - (Required) Specifies the type of Managed Identity that should be configured on this Video Analyzer. At this time the only possible value is `UserAssigned`.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Video Analyzer instance. Only possible value is `UserAssigned`.
```
Comment (jackofallops) [inline website/docs/r/video_analyzer.html.markdown:103]:
```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Video Analyzer instance. The only possible value is `UserAssigned`.
```

**Review by jackofallops — APPROVED** [review]
Thanks @stephybun. A couple of minor changes below, and a wider grammatical suggestion which may have already been discussed? ("The only" vs "Only")

Otherwise LGTM 👍

**Review by tombuildsstuff — APPROVED** [review]
A few minor comments but this otherwise LGTM 👍

**Pull request merged**